### PR TITLE
Fix BottomSheet Bug for setting Hidden State add Java APIs for (FURN android) .

### PR DESCRIPTION
**Pull request by mnkshukl**
### Platforms Impacted
- [ ] Android

### Description of changes

1. Add Fix for State.Hidden for BottomSheet.
2. Add JVMOverload for SheetItem Constructors/
3. Add a counter function for adding custom-view to bottomSheet required for FURN android.
4. Add a Scroll event hide/unhide sample for apps. 
5. Add content description API for drawer handle

### Verification
1. Open Persistent BottomSheet demo.
2. scroll down the view and bottomsheet should disappear.
3. scroll up and it should reappear.
4. All earlier functions should work as is.

### Video
BottomSheet Changes

### Pull request [checklist]

This PR has considered:
- [ NA ] Light and Dark appearances
- [ NA ] VoiceOver and Keyboard Accessibility
- [ NA ] Internationalization and Right to Left layouts
- [ NA] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt [hunk 1]
```diff
@@ -95,6 +96,7 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheet = findViewInTemplateById(R.id.persistent_bottom_sheet)!!
```
Comment (msgharpu) [inline fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:96]: Please update the version number in config.gradle as well

Comment (mnkshukl) [reply fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:96]: done

Comment (aishwarya56) [reply fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:96]: I believe merge in master for now as we have a develop branch

File: fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt [hunk 2]
```diff
@@ -238,6 +242,18 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
     }
 
+    fun hide(){
+        persistentSheetBehavior.state = BottomSheetBehavior.STATE_HIDDEN
+    }
+
+    fun show(expanded: Boolean = false) {
```
Comment (aishwarya56) [inline fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:249]: What is the difference in show(false) and hide?

Comment (mnkshukl) [reply fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:249]: States: Hidden , collapse, expanded 

show(expanded - false) -> collapsed state. -> only peek height shown

File: fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt [hunk 3]
```diff
@@ -95,6 +96,7 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheet = findViewInTemplateById(R.id.persistent_bottom_sheet)!!
         persistentSheetContainer = findViewInTemplateById(R.id.persistent_sheet_container)!!
         persistentSheetBehavior = BottomSheetBehavior.from(persistentSheet)
+        persistentSheetBehavior.isHideable = true
```
Comment (aishwarya56) [inline fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:99]: just for my info.
Why do we need this now? and why it was not used earlier?

**Review by aishwarya56 — APPROVED** [review]
Left some comments. Overall Approved

File: fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt (hunk 3 above)
Comment (mnkshukl) [reply fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt:99]: earlier code was hiding using peek height animation which is not very scalable,(like mutiple calls from an event listener etc)
this flag is required for settings state hidden

**Pull request merged**